docs/cli/edgedb_connopts.rst

```diff
@@ -23,10 +23,13 @@ Connection flags
 
 :cli:synopsis:`-I <name>, --instance=<name>`
     Specifies the named instance to connect to. The actual connection
-    parameters are stored in ``<edgedb_config_dir>/credentials`` and are
-    usually created by :ref:`ref_cli_edgedb_instance_create` or similar
-    commands. Run ``edgedb info`` to see the location of
-    ``<edgedb_config_dir>`` on your machine.
+    parameters for local and self-hosted instances are stored in
+    ``<edgedb_config_dir>/credentials`` and are usually created by
+    :ref:`ref_cli_edgedb_instance_create` or similar commands. Run ``edgedb
+    info`` to see the location of ``<edgedb_config_dir>`` on your machine.
+
+    EdgeDB Cloud instance names are in the format
+    ``<github-username>/<instance-name>``.
```

Comment:
Although we are tying orgs to Github currently, it will not necessarily be the case forever, so it's worth not to tie ourselves in the docs.  Also, it's org-name, not `username`.

```suggestion
    ``<org-name>/<instance-name>``.
```

Reply:
Thanks, @elprans! I'll update this globally.

Reply:
Got this changed globally to reflect that the org name is the first segment of the Cloud instance name.